test/check-machines-consoles

```diff
@@ -254,5 +258,76 @@ fullscreen=0
                                   "Failed when connecting: Connection closed")
 
 
+class TestMultiMachinesConsoles(ConsoleCase):
+    provision = {  # noqa: RUF012
+        "machine": {"address": "10.111.113.1/20"},
+    }
+
+    def testExternalConsole(self):
+        b = self.browser
+
+        my_ip = "10.111.113.1"
+        name = "subVmTest1"
+
+        self.setup_ssh_auth()
+        self.machine.execute(f"ssh-keyscan {my_ip} > /etc/ssh/ssh_known_hosts")
+        self.enable_multihost(self.machine)
+        self.machine.write("/etc/cockpit/cockpit.conf", "[Session]\nWarnBeforeConnecting=false\n", append=True)
+
+        self.createVm(name, graphics="vnc")
+
+        # Direct login via SSH
+
+        b.open(f"/={my_ip}/machines")
+        b.set_val('#login-user-input', "admin")
+        b.set_val('#login-password-input', "foobar")
+        b.click("#login-button")
```

Comment:
That should be `self.login_and_go("/machines", host=my_ip)`. I wonder why you don't' have to `m.start_cockpit()` here, it's usually required?

Reply:
> I wonder why you don't' have to m.start_cockpit() here, it's usually required?

Good question... I guess cockpit is enabled by default somewhere already?

Reply:
I have added a call to "start_cockpit".